openapi.yaml

```diff
@@ -45,6 +45,126 @@ tags:
       description: Find out more about Juju clouds.
       url: https://juju.is/docs/olm/manage-clouds
 paths:
+  /application/{name}:
```

Comment:
I wonder if this should be `/model/{name}/application/{name}`? Imagine you're interacting with this API using `curl` or similar, how would the server know which model you're referring to?